src/main/java/org/prebid/server/bidder/improvedigital/ImprovedigitalBidder.java

```diff
@@ -138,6 +139,10 @@ private HttpRequest<BidRequest> resolveRequest(BidRequest bidRequest, Imp imp) {
                         : null)
                 .build();
 
+        final String pathPrefix = publisherId != null && publisherId > 0
+                ? String.format("%d/", publisherId) : "";
+
+        final String endpointUrl = this.endpointUrl.replace("{PathPrefix}", pathPrefix);
```

Comment:
Pls make `"{PathPrefix}"` as a constant of class level
``` java
private static final String URL_PATH_PREFIX_MACRO = "{{PathPrefix}}";
```

Reply:
Nitpick. We prefer to have bidder macro with doubled `{{` `}}`
So it will look like `"{{PathPrefix}}"`
Also pls update yaml file with this change

Reply:
Done